src/stores/video.ts

```diff
@@ -1,10 +1,30 @@
 import { useStorage } from '@vueuse/core'
 import { defineStore } from 'pinia'
-import { reactive } from 'vue'
+import Swal from 'sweetalert2'
+import { ref } from 'vue'
 
 export const useVideoStore = defineStore('video', () => {
-  const availableIceIps = reactive<string[]>([])
+  const availableIceIps = ref<string[] | undefined>(undefined)
   const allowedIceIps = useStorage<string[]>('cockpit-allowed-stream-ips', [])
 
+  // Routine to make sure the user has chosen the allowed ICE candidate IPs, so the stream works as expected
+  const iceIpCheckInterval = setInterval(() => {
+    // Pass if there are no available IPs yet or if the user has already set the allowed ones
+    if (availableIceIps.value === undefined || !allowedIceIps.value.isEmpty()) {
+      return
+    }
+    // If there's more than one IP candidate available, send a warning an clear the check routine
+    if (availableIceIps.value.length >= 1) {
+      Swal.fire({
+        text: `Cockpit detected more than one IP being used to route the video streaming.
+          This situation often leads to video stuterring, specially if one of the IPs is from a non-wired connection.
+          To prevent issues and have an optimum streaming experience, please open the configuration of one of your
+          video widgets and choose the IP that should be used for the video streamings.`,
```

Comment:
```suggestion
        text: `Cockpit detected more than one IP being used to route the video streaming.
          This situation often leads to video stuterring, specially if one of the IPs is from a non-wired connection.
          To prevent issues and have an optimum streaming experience, please follow the steps below:
          1. open the configuration of one of your
          video widgets, and
          2. choose the IP that should be used for the video streaming.`,
```

I don't know how to format it, but the text should not be centered and should have new lines.

Reply:
By the way, shouldn't this configuration be global in cockpit over on the stream widget ?

Reply:
I liked the suggestions. Will change.

Reply:
Thanks for the notice @joaoantoniocardoso. I did that in a rush and ended up not styling the message.

This popup is going to appear for most/all users, so it should indeed look good.

Reply:
<img width="457" alt="image" src="https://github.com/bluerobotics/cockpit/assets/6551040/c039705a-3e01-401e-b544-78b7c9f7df4b">

Followed [this guide](https://www.purchase.edu/editorial-style-guide/general-style-preferences/punctuation/bulleted-and-numbered-lists/) for list formatting.